test-data/apply-configuration/overall/oauth-server-configobserver/input-dir/cluster-scoped-resources/operator.openshift.io/authentications/cluster.yaml

```
@@ -0,0 +1,14 @@
+kind: Authentication
+metadata:
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+    release.openshift.io/create-only: "true"
+  name: cluster
+spec:
+  managementState: Managed
+  observedConfig:
+    oauthServer:
+      serverArguments:
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)